pkg/sql/information_schema.go

```diff
@@ -254,7 +254,8 @@ CREATE TABLE information_schema.columns (
 	CHARACTER_SET_CATALOG    STRING,
 	CHARACTER_SET_SCHEMA     STRING,
 	CHARACTER_SET_NAME       STRING,
-	GENERATION_EXPRESSION    STRING
+	GENERATION_EXPRESSION    STRING,
+	IS_HIDDEN                STRING NOT NULL -- CockroachDB extension
```

Comment:
I have a medium-strength objection to this.  It's trouble to add things to information_schema unless we absolutely have to. I would prefer to leave information_schema.columns the same, and let the hidden-ness be ambiguous, just like it is in Postgres.